app/views/splash/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="sticky-top">
+<i class="bi bi-cash-stack"></i></div>
+<div class="shadow-lg p-3 mb-5 bg-body rounded">
+ <h1 class="text-capitalize">ExpensesCheck!</h1></div>
+ 
+ <%= link_to "Go to categories", categories_path, class: "btn btn-outline-primary" %>
+ <br>
+ <% if user_signed_in? %>
+  <p class="fw-bold"> Welcome <%= current_user.email %> ! </p>
+ <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "btn btn-outline-dark" %> 
+ <% else %>
+ <%= link_to "Sign in", new_user_session_path, class: "btn btn-outline-success" %>
+ <%= link_to 'Sign Up', new_user_registration_path, class: "btn btn-outline-success" %>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement the [previous reviewer's](https://github.com/AmaliaGM/ExpensesCheck/pull/2/files#r1056257559) suggestion: Add `Full Name` input for registration `Sign up` part 🙏🏼